### PR TITLE
Implement MyMenteesView Component for Mentor Dashboard

### PR DESCRIPTION
**Pull request by udeesha-kularathne**
## Purpose
The purpose of this PR is to address issue #20 by implementing the functionality of the MyMenteesView component.

## Goals
Implement a new component to manage and display mentee applications.

Ensure responsive design for optimal viewing on both mobile and desktop devices.

Enhance user experience by providing detailed applicant information in a user-friendly layout.

Address specific issues outlined in #20 - Mentor Dashboard - MyMenteesView

## Approach

The MyMentees component was developed to manage mentee applications.

Implemented conditional rendering to display applicant details upon selection.

## Screenshots
MyMenteesView_PR1

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: src/components/MyMenteesView/MyMentees.component.tsx [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+      </div>
+
+      {/* Right side panel showing details of selected application */}
+      <div className="w-full md:w-3/4 p-4 bg-gray-50">
+        {/* Header section with back button */}
+        <div className="flex items-center mb-10 pb-7 justify-between">
+          <div className="flex items-center">
+            <span style={{ fontSize: '48px' }}>&#8592;</span>
+            <button
+              style={{ fontSize: '28px', fontWeight: '600' }}
```
Comment (anjula-sack) [inline src/components/MyMenteesView/MyMentees.component.tsx]: Shall we use taliwind here? ex: font-md font-semibold

File: src/components/MyMenteesView/MyMentees.component.tsx [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+          ))}
+        </div>
+      </div>
+
+      {/* Right side panel showing details of selected application */}
+      <div className="w-full md:w-3/4 p-4 bg-gray-50">
+        {/* Header section with back button */}
+        <div className="flex items-center mb-10 pb-7 justify-between">
+          <div className="flex items-center">
+            <span style={{ fontSize: '48px' }}>&#8592;</span>
```
Comment (anjula-sack) [inline src/components/MyMenteesView/MyMentees.component.tsx]: Here as well

**Review by anjula-sack — CHANGES_REQUESTED** [review]

File: src/components/MyMenteesView/MyMentees.component.tsx (hunk 2 above)
Comment (udeesha-kularathne) [reply src/components/MyMenteesView/MyMentees.component.tsx]: Okay i'll change it

File: src/components/MyMenteesView/MyMentees.component.tsx [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+    );
+
+  // Function to handle click on an application
+  const handleApplicationClick = (applicant: Applicant): void => {
+    setSelectedApplication(applicant);
+  };
+
+  // Function to handle click on the back button
+  const handleBack = (): void => {
+    console.log('Back button clicked');
```
Comment (anjula-sack) [inline src/components/MyMenteesView/MyMentees.component.tsx]: Lets remove this log

**Review by anjula-sack — DISMISSED** [review]
Requested a small change

File: src/components/MyMenteesView/MyMentees.component.tsx (hunk 3 above)
Comment (udeesha-kularathne) [reply src/components/MyMenteesView/MyMentees.component.tsx]: Okay. i removed it

**Comment by anjula-sack** [comment]
Good job @udeesha-kularathne!

**Pull request merged**